src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/JtsMvt.java

```diff
@@ -7,14 +7,16 @@
  */
 public class JtsMvt {
 
-    /** Map layers by name */
+    /**
+     * Map layers by name.
+     */
     private final Map<String, JtsLayer> layersByName;
 
     /**
      * Create an empty MVT.
      */
     public JtsMvt() {
-        this(Collections.emptyList());
+        this(Collections.<JtsLayer>emptyList());
```

Comment:
Will this work in android? I noticed we were taking out the other emptyList() calls.

Reply:
We should be fine with Java 7/8 syntax - the problem rather with [missing functionality](https://developer.android.com/studio/write/java8-support.html#supported_features) at the language level. 👍